user/templates/users.html

```diff
@@ -1,4 +1,5 @@
 {% extends 'base_user.html' %}
+{% load static %}
```

Comment:
toto mozes mat kludne v base_user.html a nemusis to pisat znova, ale to neni chyba len makes life easier